src/renderer/views/Hashtag/Hashtag.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div>
+    <ft-loader
+      v-if="isLoading"
+      :fullscreen="true"
+    />
+    <div v-else>
+      <ft-card>
+        <h3>{{ hashtag }}</h3>
+      </ft-card>
+      <ft-card>
```

Comment:
Should probably put the title and the content in the same card, so it matches the design of the rest of FreeTube
```suggestion
```